xml/deployment_ganesha.xml

```diff
@@ -408,6 +408,105 @@ Full list of resources:
    </procedure>
   </sect2>
 
+  <sect2 xml:id="setup-portblock-resource">
+    <title>Setting Up PortBlock Resource</title>
+    <para>When a service goes down, the TCP protocol that is in use by
+      nfs-&ganesha; is required to be closed by the user, otherwise it
```

Comment:
we could omit "by the user", we just need to close it, here we do it from the server side.